src/solver/toolkit.c

```diff
@@ -1874,6 +1874,8 @@ int DLLEXPORT swmm_getNodeResult(int index, SM_NodeResult type, double *result)
                             + Node[index].invertElev) * UCF(LENGTH); break;
             case SM_LATINFLOW:
                 *result = Node[index].newLatFlow * UCF(FLOW); break;
+	    case SM_HRT:
+		*result = Node[index].hrt; break;
```

Comment:
Is SM_HRT an input parameter or a result of a calculation?

Reply:
HRT is computed by the SWMM and this exposes the computed HRT to users.